components/filemanager/context-menu.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Context Menu
+page_title: FileManager Context Menu
+description: Context Menu of the FileManager for Blazor.
+slug: filemanager-context-menu
+tags: telerik,blazor,filemanager,contextmenu, menu, context
+published: True
+position: 3
```

Comment:
(not urgent right now) It's better to assign positions with free spots in between - 10, 15, 20, etc. This allows easier insertion of new articles in the future.